docs/maintenance.md

```diff
@@ -0,0 +1,81 @@
+Maintenance guide
+=================
+
+This is the maintenance guide for pvc-autoresizer.
```

Comment:
This line is not necessary because it's clear that this document is for maintenance by the document title.

Reply:
done.